src/init/ossec-server.sh

```diff
@@ -219,6 +219,7 @@ start()
             else
                 echo "${i} already running..."
             fi
+        fi
```

Comment:
This change will be superseded by commit a48989506d8e1518a0bab5f313c4f283b94cb081 (sorry about that)